docs-styleguide.md

```diff
@@ -5,6 +5,7 @@ This is the documentation styleguide for our natural language descriptions used
 ## Code references
 
 * Use `backticks` for every code example inside of a normal description. In the CLI, where you cannot use backticks, use 'single quotes' for commands.
+* For bash arguments, do not include a `$` before commands in man pages. A `$` may be included in READMEs.
```

Comment:
I'm partial to `command line`, if only because I get the [subjective] impression more people know what it means than `shell`. Either works for me. :+1: